### PR TITLE
GiftCard Transaction Settlement updated

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to update description of the Create GiftCard Transaction Settlement endpoint (EDU-9757)

### CODE REVIEW

File: VTEX - Giftcard Provider Protocol.json [hunk 1]
```diff
@@ -1107,9 +1107,9 @@
 						}
```
Comment (julia-rabello) [inline VTEX - Giftcard Provider Protocol.json:1107]: Na linha 1106, acho que vale remover o `default` e incluir um `example` com um exemplo de valor real para esse parâmetro.

File: VTEX - Giftcard Provider Protocol.json [hunk 2]
```diff
@@ -1107,9 +1107,9 @@
...
 					},
 					{
-						"name": "transactionID",
+						"name": "tID",
 						"in": "path",
-						"description": "",
+						"description": "Transaction ID generated by the provider. It is different from the `transactionID` generated by VTEX.",
 						"required": true,
 						"style": "simple",
 						"schema": {
```
Comment (julia-rabello) [inline VTEX - Giftcard Provider Protocol.json:1115]: Na linha 1117, acho que vale remover o `default` e incluir um `example` com um exemplo de valor real para esse parâmetro.

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: VTEX - Giftcard Provider Protocol.json [hunk 3]
```diff
@@ -2375,7 +2385,6 @@
 				}
 			},
 			"response4": {
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard Provider Protocol.json:2387]: Boa! Modificado.

File: VTEX - Giftcard Provider Protocol.json (hunk 1 above)
Comment (ricardoaerobr) [reply VTEX - Giftcard Provider Protocol.json:1107]: Corrigido.

File: VTEX - Giftcard Provider Protocol.json (hunk 2 above)
Comment (ricardoaerobr) [reply VTEX - Giftcard Provider Protocol.json:1115]: Corrigido.

**Review by karenkrieger — DISMISSED** [review]
LGTM

**Review by julia-rabello — DISMISSED** [review]

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**